host: "192.168.1.191"
port: 8081
posthost: "192.168.1.191"
postport: 8081
timeout: 20
post_addr: postaddr
state_topic: state
task_topic: task
det_topic: det